forks/driver-toolkit/Dockerfile.centos9

```diff
@@ -0,0 +1,71 @@
+FROM registry.ci.openshift.org/ocp/4.12:base
```

Comment:
is this image being replaced somehow? Otherwise I'd suggest to simply use a CentOS Stream image as base here (and everywhere else in this repo). As an optimization, we might want to create minimal stream container later on, too.

Reply:
Yes, in buildconfigs we set
```
      from:
        kind: "ImageStreamTag"
        name: "release:base"
```

Reply:
It's replaced by the buildconfig as of now. Also, I wanted to do minimal and backward-compatible changes as we may want to open a PR to the "upstream" driver-toolkit eventually (I'm not sure @vrutkovs, is that the intention of the "forked dockerfiles?").

Reply:
>we may want to open a PR to the "upstream" driver-toolkit repo, as an example, eventually

yes, exactly